src/Auth0Client.ts

```diff
@@ -128,13 +133,18 @@ export default class Auth0Client {
   private tokenIssuer: string;
   private defaultScope: string;
   private scope: string;
+  private CookieStorage: ClientStorage;
```

Comment:
What's the reason for making this Title Case? Could we follow convention for other class-level props?

Reply:
I was generally using the convention `SomethingTitleCase = SomeOtherThingTitleCase || YetAnotherThingTitleCase;` - but sure, I can change to camel case